src/Config.js

```diff
@@ -78,11 +78,11 @@ module.exports = class extends ChainedMap {
 
         // improve rule/use output
         if (value && value.__ruleNames) {
-          const prefix = `/* ${configPrefix}.module.rule('${
-            value.__ruleNames[0]
-          }')${value.__ruleNames
-            .slice(1)
-            .map(r => `.oneOf('${r}')`)
+          const ruleTypes = value.__ruleTypes;
+          const prefix = `/* ${configPrefix}.module${value.__ruleNames
+            .map(
+              (r, index) => `.${ruleTypes ? ruleTypes[index] : 'rule'}('${r}')`,
```

Comment:
Do we need the default value here given that `src/Rule.js` sets the default type to `'rule'`?

Reply:
I added it as a fail safe in case someone was messing with manually generating `__ruleNames`, but it's not really necessary. I can remove it if you don't think it's necessary.

Reply:
Yeah let's leave for now